praw/exceptions.py

```diff
@@ -47,6 +47,36 @@ def __init__(self):
         )
 
 
+class InvalidImplicitAuth(ClientException):
+    """Indicate exceptions where an implicit auth type is used incorrectly."""
+
+    def __init__(self):
+        """Instantize the class."""
+        super().__init__(
+            "Implicit authorization can only be used with installed apps."
+        )
+
+
+class InvalidURL(ClientException):
+    """Indicate exceptions where an invalid URL is entered."""
+
+    def __init__(self, url: str, custom_message: Optional[str] = None):
+        """Instantize the class.
+
+        :param url: The invalid URL.
+        :param custom_message: A custom message to display. Must contain a
+            format identifier (``{}`` or ``{0}``).
+        """
+        if custom_message:
+            super().__init__(custom_message.format(url))
+        else:
+            super().__init__("Invalid URL: {url}".format(url=url))
```

Comment:
Can you make `Invalid URL: {}` the default `custom_message` so that this conditional is not needed?